### PR TITLE
clang-format Global and headers of viz

### PR DESCRIPTION
**Pull request by angriman**
This is a continuation of #803

### CODE REVIEW

File: include/networkit/viz/GraphLayoutAlgorithm.hpp [hunk 1]
```diff
@@ -124,18 +126,19 @@ class GraphLayoutAlgorithm {
...
+            file << "{a}" << vertexCoordinates[u][0] << " " << vertexCoordinates[u][1] << " "
+                 << vertexCoordinates[u][2] << std::endl;
         });
 
         // edges
         file << std::endl;
         file << "@subgroup {edges} dominant" << std::endl;
         file << "@vectorlist {edges} color= white" << std::endl;
         G->forEdges([&](node u, node v) {
-            //if (u <= v) { // draw graph undirected
```
Comment (fabratu) [inline include/networkit/viz/GraphLayoutAlgorithm.hpp:135]: Does the formatting automatically remove the comment here? In this case it more likely an artifact (so it's fine I think), but in general deleting comments by formatting should be avoided.

Comment (angriman) [reply include/networkit/viz/GraphLayoutAlgorithm.hpp:135]: No, I thought it was ok to remove unused code so I did it, but I can move this change to another PR if necessary.

**Review by fabratu — APPROVED** [review]
LGTM.

**Review by avdgrinten — COMMENTED** [review]
Please keep manual changes out of automatic reformatting commits.

**Review by fabratu — APPROVED** [review]

**Comment by angriman** [comment]
I moved the manual changes to a separate commit, we can merge this now.

**Pull request merged**